nixos/modules/services/web-apps/jitsi-meet.nix

```diff
@@ -293,7 +295,7 @@ in
         enabled = true;
         domain = cfg.hostName;
         extraConfig = ''
-          authentication = "jitsi-anonymous"
+          authentication = ${if cfg.secureDomain.enable then "\"internal_hashed\"" else "\"jitsi-anonymous\""}
```

Comment:
https://jitsi.github.io/handbook/docs/devops-guide/secure-domain/ refers to `"anonymous"` rather than `"jitsi-anonymous"`

Reply:
Hi, but I took this from their example file on GitHub: https://github.com/jitsi/jitsi-meet/blob/master/doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example.
I have no idea if that makes a difference, they seem to talk about it a little in: https://wiki.archlinux.org/title/Talk:Jitsi-meet.